src/chains/ethereum/transaction/package.json

```diff
@@ -48,17 +48,17 @@
     "truffle"
   ],
   "dependencies": {
-    "@ethereumjs/common": "2.6.4",
+    "@ethereumjs/common": "2.6.5",
     "@ganache/ethereum-address": "0.5.0",
     "@ganache/ethereum-utils": "0.5.0",
     "@ganache/rlp": "0.5.0",
     "@ganache/secp256k1": "0.4.0",
     "@ganache/utils": "0.5.0",
-    "ethereumjs-util": "7.1.4",
-    "node-gyp-build": "4.2.3"
+    "ethereumjs-util": "7.1.5",
+    "node-gyp-build": "4.5.0"
```

Comment:
`@ethereumjs/*` packages have updated to this version of `node-gyp-build`, so I matched our dependency with theirs so we don't have to install multiple.